### ISSUE TITLE
Request for "natural" Obsidian tag

### ISSUE THREAD

**Issue by ghost** [neoforged/NeoForge#1548]
# Background
I have been using the `c:obsidians` tag in Expanded Storage for recipes like for my `Obsidian Chest` however I recently noticed #1536 which added `Crying Obsidian` to the tag which was not expected on my part.

*This is where I had planned to talk about modded glass blocks and how they can exist in the `c:glass_blocks` tag but not the `c:glass_blocks/cheap` tag but turns out very few mods actually add their blocks to these tags...*

In any case, there's a cheap glass blocks tag to target blocks without advance behaviour. So it would include Vanilla's `Glass` block and `Stained Glass` blocks but not the `Tinted Glass` block.
> Glass which is made from cheap resources like sand and only minor additional ingredients like dyes.

# Proposal
Basically I'd like a tag for obsidian blocks without materials infused into it and without unique behaviours, probably can't call them cheap like glass maybe something like `c:obsidians/natural`? Probably wouldn't be technically correct either...

*sorry if this is way too wordy....*

**Comment by TelepathicGrunt** [comment]
Probably
c:obsidians/normal 
c:obsidians/crying

Crying obsidians spawns in ruined portals and in bastion loot so it could be viewed as natural generation.

**Comment by HenryLoenwind** [comment]
How about a tag for all "just like the simple and cheap vanilla block/item, a perfect alternative input for crafting" stuff? Maybe also a "this is the actual vanilla block/item" and a "works the same as the vanilla stuff, but you probably won't want to use it as a substitute for crafting" tag.

I mean, it works for the coloured stuff, although those tags have way better names...

**Comment by KnightMiner** [comment]
I feel like a lot of these tag discussions keep coming back to the tags need their exact usage documented if they are going to be controlled by NeoForge. Some people assume an obsidian is "vanilla obsidian but a different variant because whatever cosmetic or gameplay reason its different", while some thing it means "anything that includes obsidian in the name".

If a tag has no javadoc, we should not be changing which vanilla blocks it contains short of a new vanilla update coming without first adding a javadoc.
